src/NexusMods.App/App.axaml

```diff
@@ -12,6 +12,7 @@
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <ResourceInclude Source="avares://NexusMods.App.UI/Pages/LibraryPage/TreeDataGridLibraryResources.axaml" />
+                <ResourceInclude Source="avares://NexusMods.App.UI/Pages/Sorting/LoadOrder/TreeDataGridLoadOrderResources.axaml" />
```

Comment:
The reason `TreeDataGridLibraryResources` is here is because those resources need to be available to multiple TreeDataGrids (Library, Collections, Loadout).

In the case of LoadOrder, I don't think we need them shared, so we can have this import directly in the LoadOrderView.axaml file.
This way they are easier to find and we don't run the risk of the templates being used in the wrong place.